docsrc/conf.py

```diff
@@ -18,7 +18,7 @@
 import os
 import sys
 
-sys.path.insert(0, os.path.abspath('.'))
+sys.path.insert(0, os.path.abspath('..'))
```

Comment:
Why the change here? This was consistent across all our libraries using sphinx.

Reply:
Yep, I noticed this is consistent across the libraries. I am not sure why it gives me an error in this case. If I use `os.path.abspath('.')`, I got an AttributeError when I do `make github`. It happens when I import class `DiversityRecoMetrics` from module `jurity.recommenders`. So it can't find the module. Maybe there is some setup not right in my env? Any idea?

Reply:
I run the command in `docsrc` directory.

Reply:
Had a double check with @dorukkilitcioglu. Both `.` and `..` are working fine if installed in editable mode. If pip installed Jurity, using `..` would work then. Given `.` seems to add the current path of `conf.py` to `sys.path`, this line seems not useful though. So keeping it as `..` might make sense.

Reply:
I confirm I can generate the documentation and it displays fine with `..` when running `make github` inside `docsrc`